src/Compilers/CSharp/Portable/Compilation/MemberSemanticModel.cs

```diff
@@ -701,6 +681,21 @@ internal override LocalSymbol GetAdjustedLocalSymbol(LocalSymbol local, int posi
 
         private LocalFunctionSymbol GetDeclaredLocalFunction(LocalFunctionStatementSyntax declarationSyntax, SyntaxToken declaredIdentifier)
```

Comment:
>declaredIdentifier [](start = 121, length = 18)

Can we get this from ```declarationSyntax```?

Reply:
Yes, but I was avoiding changing existing code as much as I could here.

---
In reply to: [327350060](https://github.com/dotnet/roslyn/pull/38769#discussion_r327350060) [](ancestors = 327350060)

Reply:
> Yes, but I was avoiding changing existing code as much as I could here.

It looks like there is only one consumer and it passes ```declarationSyntax.Identifier```. Consider getting that below where we actually need it.

---
In reply to: [327351863](https://github.com/dotnet/roslyn/pull/38769#discussion_r327351863) [](ancestors = 327351863,327350060)